### PR TITLE
improve FORM_ERROR_MARKER to better match PFG

### PR DESCRIPTION
**Pull request by wobsta**
see discussion at pull request #238

### CODE REVIEW

File: src/collective/easyform/actions.py [hunk 1]
```diff
@@ -513,6 +514,13 @@ def __init__(self, **kw):
         super(CustomScript, self).__init__(**kw)
 
     def getScript(self, context):
+        # for the convenience of scripters operating
+        # in a restricted python environment,
+        # let's store a reference to FORM_ERROR_MARKER
+        # on the object, so it'll be available
+        # as an attribute of context.
+        context.FORM_ERROR_MARKER = FORM_ERROR_MARKER
```
Comment (mauritsvanrees) [inline src/collective/easyform/actions.py:522]: `context` here is the easyform object. This means that every time someone fills in the form, a database change is made. Maybe make it available on the request instead?

    context.REQUEST.FORM_ERROR_MARKER = FORM_ERROR_MARKER

Comment (wobsta) [reply src/collective/easyform/actions.py:522]: I thought about this as well, but was not sure and did not check whether this results in a DB change each time. We certainly should not do that. Sorry.

I suggested and did the change at the context, as this is how the FORM_ERROR_MARKER is done at PFG. OTOH I am fine to use the request instead. Let me quickly update my fork accordingly ...

**Comment by wobsta** [comment]
This pull request should not be responsible to the travis error. It failure is due to some other commits yesterday. (The doctests run fine locally, which I just re-confirmed.)

**Comment by mauritsvanrees** [comment]
I have triggered a Travis build on master. See also [this comment](https://github.com/collective/collective.easyform/pull/239#issuecomment-675354826). I will wait till that one finishes, but I expect the same failure there.

Code looks good now. Recapping how it works after your PR from my testing, for good measure:

- `return {"random": "Oh noes!"}`: only the general form error 'please check all fields' error is displayed.
- `return {"topic": "Oh noes!"}`: general form error, plus the topic field shows the given error.
- `return {request.FORM_ERROR_MARKER: "Oh noes!"}`: general form error shows the given message

Seems good. Thanks!
Waiting for the master build to finish now.

**Comment by mauritsvanrees** [comment]
Same error as on master now. I merge.

**Pull request merged**